src/useBreakpoints.js

```diff
@@ -14,6 +14,8 @@ const boxOptions = {
  * @returns {*} Value of `breakpoints` at matching breakpoint.
  */
 const findBreakpoint = (breakpoints, entrySize) => {
+  if (typeof entrySize === 'undefined') return undefined;
```

Comment:
Would this be simpler? do we have to check the type for some reason i'm not thinking of?

```suggestion
  if (entrySize === undefined) return undefined;
```

Also do you have tests for the package? should some test be updated?

Reply:
* Yes, it would be simpler and I suppose it's a muscle-memory to check for undefined like that because "`undefined` can be redefined". Since I'm abstracting away code for package users, I don't see a problem with using the long-winded way to increase robustness ever so slightly.
* I'm looking into adding tests. No tests currently, so none to update.